### PR TITLE
[REVIEWS-27] Instructions for the upgrade from to v2 to v3

### PR DESCRIPTION
**Pull request by Arturo777**
#### What problem is this solving?
Help to the users to upgrade review and ratings app to upgrade to v3 from v2, adding some steps on the documentation.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ Reviews & Ratings is a VTEX IO native solution that allows shoppers to submit re
 
 ### Step 1 - Installing the Reviews and Ratings app
 
-Using your terminal, log in to the desired VTEX account and run the following command:
+Using your terminal, log in to the desired VTEX account and run the following commands:
 
-`vtex install vtex.reviews-and-ratings@2.x`
+`vtex install vtex.reviews-and-ratings@3.x`
+
+> ℹ️ _If you have already installed  `vtex.reviews-and-ratings@2.x` **you have to migrate the reviews to master data** by making this `GET` call: `https://{accountName}.{environment}.com/admin/reviews-ratings/migrate-data`, after that you can uninstall the version 2 by runing the following command: `vtex uninstall vtex.reviews-and-ratings@2.x`_
```
Comment (btalma) [inline docs/README.md:23]: Please remove `/admin` from the migrate url

Comment (btalma) [reply docs/README.md:23]: https://{accountName}.{environment}.com/reviews-and-ratings/migrate-data

Comment (Arturo777) [reply docs/README.md]: Immediately

**Review by btalma — DISMISSED** [review]

**Review by albertm805 — DISMISSED** [review]
LGTM

**Pull request closed without merging**